experimental/dds/tree2/src/feature-libraries/default-field-kinds/defaultChangeFamily.ts

```diff
@@ -86,9 +89,8 @@ export interface IDefaultEditBuilder {
 	/**
 	 * Moves a subsequence from one sequence field to another sequence field.
 	 *
-	 * Note that the `destinationIndex` is the final index the first node moved should end up at.
-	 * Thus if moving nodes from a lower to a higher index within the same field, the `destinationIndex` must be computed as if the subsequence being moved has already been removed.
-	 * See {@link SequenceFieldEditBuilder.move} for details.
+	 * Note that the `destinationIndex` is interpreted based on the state of the sequence *before* the move operation.
+	 * For example, `move(field, 0, 2, field, 1)` changes `[A, B, C]` to `[B, A, C]`.
```

Comment:
Doesn't this call move A and B to a location in between A and B? Shouldn't the result still be `[A, B, C]`.

Reply:
Yeah, I totally messed that up.